### DISCUSSION TITLE
set-up sellers for my products

### DISCUSSION THREAD

**Discussion by asimasiddiqui** [vendurehq/vendure#1803] [Q&A]
Hi, I've read about channels and multi-tenant store fronts. However, I don't need to assign products to different channels or storefronts. The products will all stay in one channel, but i want to represent each seller with a code/QR-code, etc which I will then use to route payment settlements. 

How can I achieve this? any pointers.

**Comment by michaelbromley** [comment]
Hi!
You could create a new entity to represent the seller, and then I think you'll need a custom PaymentMethodHandler implementation if you want to split the payments. But that's not something I've done before so I'm afraid I can't offer concrete advice on exactly how it should work.
